### PR TITLE
update a bunch of dependencies to fix a few problems with git

### PR DESCRIPTION
**Pull request by PhilTaken**

### CODE REVIEW

**Comment by PhilTaken** [comment]
@rvem this builds locally for me, not sure what the buildkite error is about

**Comment by rvem** [comment]
Looks like a temporary issue that was resolved by retry :thinking:

**Comment by rvem** [comment]
But now there is another error with `libgit2-sys`:(

**Comment by rvem** [comment]
> It appears building the package is not part of nix flake check

:coolstory:

File: flake.nix [hunk 1]
```diff
@@ -9,55 +9,65 @@
...
 
         checks = {
           trailing-whitespace = pkgs.build.checkTrailingWhitespace ./.;
           reuse-lint = pkgs.build.reuseLint ./.;
+          inherit update-daemon;
         };
 
         devShell = pkgs.mkShell {
-          RUST_LOG = "trace";
+          #RUST_LOG = "trace";
```
Comment (rvem) [inline flake.nix:70]: Looks like a leftover

**Review by rvem — APPROVED** [review]

**Comment by rvem** [comment]
Don't forget to squash fixups

**Comment by rvem** [comment]
This resolve the issue, so I'm merging it already:)

**Pull request merged**